callbacks_six.go

```diff
@@ -0,0 +1,44 @@
+// +build six
+
+package alpm
+
+/*
+#cgo CFLAGS: -DSIX=1
+#include "callbacks_six.h"
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+type (
+	logCallbackSig      func(LogLevel, string)
```

Comment:
This should be a struct containing the function and the context we want to pass in. So we can do:

```go
func my_log(LogLevel, string, ctx Interface) {
    //stuff
}

handle.SetLogCallback(my_log, value)
```